### PR TITLE
NOISSUE - Update Docker compose

### PR DESCRIPTION
**Pull request by dborovcanin**

### CODE REVIEW

File: http/swagger.yaml [hunk 1]
```diff
@@ -43,6 +43,8 @@ paths:
           description: Message is accepted for processing.
         400:
           description: Message discarded due to its malformed content.
+        401:
```
Comment (mijicd) [inline http/swagger.yaml]: This is wrong and misleading. 401 should be thrown exclusively on missing credentials (token). Permissions issues should be taken care via 403. Please review the other API docs as well.

File: docker/docker-compose.yml [hunk 2]
```diff
@@ -63,33 +48,88 @@ services:
...
+    ports:
+      - 8180:8180
+
+  coap:
+    image: mainflux/coap:latest
+    container_name: mainflux-coap
+    expose:
+      - 5683
+    restart:
+      on-failure
```
Comment (mijicd) [inline docker/docker-compose.yml]: Nitpick: remove linebreak: `restart: on-failure`.

File: docker/docker-compose.yml [hunk 3]
```diff
@@ -35,25 +35,10 @@ services:
...
-    restart:
-      on-failure
-    environment:
-      MF_DB_HOST: postgres
-      MF_MANAGER_DB: mainflux
-      MF_MANAGER_PORT: 8180
-      MF_MANAGER_SECRET: test-secret
+        - 5432:5432
+    volumes:
+    - ./init.sql:/docker-entrypoint-initdb.d/init.sql
```
Comment (mijicd) [inline docker/docker-compose.yml]: Please drop this and spawn 2 DB containers.

**Review by mijicd — CHANGES_REQUESTED** [review]

File: docker/nginx.conf [hunk 4]
```diff
@@ -126,62 +119,56 @@ http {
...
-        ssl_session_cache shared:SSL:10m;
-        ssl_session_tickets off;
+    # Proxy pass to mainflux-http-adapter
+    location /http/ {
+        proxy_redirect off;
+        proxy_set_header Host $host;
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Proto $scheme;
+        proxy_pass http://http-adapter:8185/;
```
Comment (nmarcetic) [inline docker/nginx.conf:164]: Remove this `/` at the end.

File: docker/docker-compose.yml [hunk 5]
```diff
@@ -27,69 +26,118 @@ services:
...
+    restart: on-failure
+    environment:
+      POSTGRES_USER: mainflux
+      POSTGRES_PASSWORD: mainflux
+      POSTGRES_DB: clients
+
+  clients:
+    image: mainflux/clients:latest
+    container_name: mainflux-clients
+    depends_on:
```
Comment (nmarcetic) [inline docker/docker-compose.yml:73]: No deps to `clients-db`  ?

File: docker/docker-compose.yml [hunk 6]
```diff
@@ -27,69 +26,118 @@ services:
...
+      MF_USERS_DB_PORT: 5432
+      MF_USERS_DB_USER: mainflux
+      MF_USERS_DB_PASS: mainflux
+      MF_USERS_DB: users
+      MF_USERS_HTTP_PORT: 8180
+      MF_USERS_GRPC_PORT: 8181
+      MF_USERS_SECRET: secret
+    ports:
+      - 8180:8180
+
```
Comment (nmarcetic) [inline docker/docker-compose.yml:60]: No deps to `users-db` ?

**Review by nmarcetic — DISMISSED** [review]

**Review by mijicd — APPROVED** [review]

**Pull request merged**